core/src/test/java/org/jdbi/v3/core/TestJsr310.java

```diff
@@ -35,6 +36,8 @@
 
     Handle h;
 
+    Clock fixed = Clock.fixed(Instant.ofEpochSecond(123456789), ZoneOffset.UTC);
```

Comment:
I'd like to see the reason for this clock being here (incompatible precisions) documented, either in a comment or even better through its name, like `withPgPrecision`